ext/datasource/etcdv3/etcdconfig.go

```diff
@@ -0,0 +1,38 @@
+package etcdv3
+
+import (
+	"github.com/alibaba/sentinel-golang/core/config"
+	"strings"
+)
+
+const(
+	EndPoints = "endpoints"
```

Comment:
The key might need to keep consistent with Java version.
refer to [https://github.com/alibaba/Sentinel/blob/master/sentinel-extension/sentinel-datasource-etcd/src/main/java/com/alibaba/csp/sentinel/datasource/etcd/EtcdConfig.java](https://github.com/alibaba/Sentinel/blob/master/sentinel-extension/sentinel-datasource-etcd/src/main/java/com/alibaba/csp/sentinel/datasource/etcd/EtcdConfig.java)

Reply:
accept